packages/clerk-js/src/ui.retheme/components/OrganizationProfile/ActiveMembersList.tsx

```diff
@@ -76,9 +76,9 @@ const MemberRow = (props: {
   const { membership, onRemove, onRoleChange, options } = props;
   const { localizeCustomRole } = useLocalizeCustomRoles();
   const card = useCardState();
-  const user = useCoreUser();
+  const { user } = useUser();
```

Comment:
❓ Don't we need  `if (!user) return null;` guard here?

Reply:
OrganizationProfile is wrapped in  `withCoreUserGuard` so we are safe.